systemtest/src/test/java/io/strimzi/systemtest/metrics/MetricsST.java

```diff
@@ -385,6 +391,62 @@ void testCruiseControlMetrics() {
         }
     }
 
+    @Test
+    void testKafkaMetricsSettings() {
```

Comment:
I think the test should test the actual metrics. I think you should also consider:
* Having the test for all affected resources
* Test things such as updates to the metrics
* Eventually migrations from one set of metrics to another